packages/wrangler/src/__tests__/ai.test.ts

```diff
@@ -117,6 +117,10 @@ describe("ai commands", () => {
 	});
 
 	it("should truncate model description", async () => {
+		const original = process.stdout.columns;
+		// Arbitrary fixed value for testing
```

Comment:
I picked this because it meant the snapshot could stay the same. It's an entirely arbitrary choice though

Reply:
Clicked through to the diff expecting a round number – wasn't expecting... 186! 😆